frontends/web/src/routes/buy/moonpay.tsx

```diff
@@ -85,7 +85,7 @@ class Moonpay extends Component<Props, State> {
 
     public render() {
         const { moonpay, t } = this.props;
-        const { height } = this.state;
+        const { height } = this.state || { height: 500 }; //Check why state is null (Load HOC?)
```

Comment:
On first render `this.state` is null, looks strange for me, must be something about the Load HOC. Still the fix works as intended, and on first re-render the height state is fixed

Reply:
This should be fixed with setting `state = {}` on the body, lets add it later